client/service_inline.go

```diff
@@ -65,7 +65,7 @@ type serviceInlineClient struct {
 type ServerInitialInfo interface {
 	Endpoints() endpoint.Endpoint
 	Option() *internal_server.Options
-	GetServiceInfo() *serviceinfo.ServiceInfo
+	GetServices() map[string]*serviceinfo.Service
```

Comment:
Breaking Change: Need to be compatible with this repo: [repo](https://github.com/kitex-contrib/opensergo/blob/47baeb126c27cfcc8f821f5e73c3c4f353051d59/example/server/main.go#L48)

Reply:
@jayantxie So you mean the function must be the same (`GetServiceInfo() *serviceinfo.ServiceInfo`), right?